libkpod/config.go

```diff
@@ -0,0 +1,267 @@
+package libkpod
```

Comment:
Okay, I can remove this.  With that being said, is there anything in the default config that will change since the default config now applies to more than just cri-o?  For example, should the logdir still be /var/log/crio or should it be something more generic like /var/log/containers?